site-src/index.md

```diff
@@ -94,8 +94,32 @@ This flexibility allows the API to adapt to vastly different
 organizational models and implementations while remaining a portable and
 standard API.
 
+## What's the difference between Gateway API and an API Gateway?
 
-## Who is working on Gateway?
+The concept of a Gateway is a networking component that allows data to transfer from one
+network to another network.
+
+An API Gateway is a Gateway implementation that deals more specifically with backend services
+and routes client reqeusts to those backend services. Those services expose an
+API such as HTTP, REST or some other protocol on top of TCP or UDP.
```

Comment:
How about something like this?

```suggestion
An API Gateway is a general concept that describes anything that exposes capabilities of a backend service, while providing extra capabilities for traffic routing and manipulation, such as load balancing, request and response transformation, and sometimes more advanced features like authentication and authorization, rate limiting, and circuit breaking.

Most Gateway API implementations are API Gateways to some extent, but not all API Gateways are Gateway API implementations.
```

The last line may be better after the explanation of Gateway API.